mealie/repos/seed/resources/foods/locales/es-ES.json

```diff
@@ -1,222 +1,692 @@
+    },
+    "butternut-pumpkin": {
+        "name": "calabaza cacahuete"
+    },
+    "butternut-squash": {
+        "name": "calabaza moscada"
+    },
+    "cabbage": {
+        "name": "repollo",
+        "plural_name": "cabbages"
+    },
+    "cactus-edible": {
+        "name": "higo chumbo"
+    },
+    "calabrese": {
+        "name": "br\u00f3coli"
+    },
+    "cane-sugar": {
+        "name": "az\u00facar de ca\u00f1a"
+    },
+    "cannabis": {
+        "name": "cannabis"
+    },
+    "capsicum": {
+        "name": "\u00f1ora"
+    },
+    "caraway": {
+        "name": "comino"
+    },
+    "carrot": {
+        "name": "zanahoria",
+        "plural_name": "carrots"
+    },
+    "caster-sugar": {
+        "name": "az\u00facar de ca\u00f1a"
+    },
+    "castor-sugar": {
+        "name": "az\u00facar"
+    },
+    "catfish": {
+        "name": "siluro"
+    },
+    "cauliflower": {
+        "name": "coliflor",
+        "plural_name": "cauliflowers"
+    },
+    "cayenne-pepper": {
+        "name": "pimienta de cayena"
+    },
+    "celeriac": {
+        "name": "apio nabo"
+    },
+    "celery": {
+        "name": "apio"
+    },
+    "cereal-grains": {
+        "name": "cereales en grano"
+    },
+    "chard": {
+        "name": "acelgas"
+    },
+    "cheese": {
+        "name": "queso"
+    },
+    "chicory": {
+        "name": "achicoria"
+    },
+    "chilli-peppers": {
+        "name": "chile/guindilla",
+        "plural_name": "chilli peppers"
+    },
+    "chinese-leaves": {
+        "name": "col china"
+    },
+    "chives": {
+        "name": "cebollino"
+    },
+    "chocolate": {
+        "name": "chocolate"
+    },
+    "cilantro": {
+        "name": "cilantro"
+    },
+    "cinnamon": {
+        "name": "canela"
+    },
+    "clarified-butter": {
+        "name": "mantequilla clarificada"
+    },
+    "coconut": {
+        "name": "coco",
+        "plural_name": "coconuts"
+    },
+    "coconut-milk": {
+        "name": "leche de coco"
+    },
+    "cod": {
+        "name": "bacalao"
+    },
+    "coffee": {
+        "name": "caf\u00e9"
+    },
+    "collard-greens": {
+        "name": "berza"
+    },
+    "confectioners-sugar": {
+        "name": "az\u00facar"
+    },
+    "coriander": {
+        "name": "cilantro"
+    },
+    "corn": {
+        "name": "ma\u00edz",
+        "plural_name": "corns"
+    },
+    "corn-syrup": {
+        "name": "jarabe de ma\u00edz"
+    },
+    "cottonseed-oil": {
+        "name": "aceite de algod\u00f3n"
+    },
+    "courgette": {
+        "name": "calabac\u00edn"
+    },
+    "cream-of-tartar": {
+        "name": "cr\u00e9mor t\u00e1rtaro"
+    },
+    "cucumber": {
+        "name": "pepino",
+        "plural_name": "cucumbers"
+    },
+    "cumin": {
+        "name": "comino"
+    },
+    "daikon": {
+        "name": "daikon",
+        "plural_name": "daikons"
+    },
+    "dairy-products-and-dairy-substitutes": {
+        "name": "productos l\u00e1cteos y sustitutos de l\u00e1cteos"
```

Comment:
FYI, looks like there's encoding issues from however you transformed these.

Reply:
This should be resolved now